fastlane_core/lib/fastlane_core/env.rb

```diff
@@ -0,0 +1,12 @@
+module FastlaneCore
+  class Env
+    def self.enabled?(env)
+      return false unless ENV[env]
+      unless ENV["SKIP_ENV_HANDLER"]
```

Comment:
ok i dropped the idea. initially i thought of encapsulate all `ENV` calls - and there would be a way to use an explicit `0` as a value.

but since its a true/false only thing now, i think its good to remove.